### PR TITLE
Feature/translation w3

### PR DESCRIPTION
**Pull request by webanimal**
* W3 translated;
* Added new helper for Hot Keys;
* Readme updated.

### CODE REVIEW

File: src/main/kotlin/day_1_kotlin/langeng/info/D_HowToUseHotkeys.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    * Intellij idea and Android studio have a lot of templates, shortcuts and hot keys.
+    * And is very flexible and customizable. There are several of them:
+    * */
+
+    // For a fast jump to a method or variable,
+    // place a cursor on it and press "Ctrl+B"
+    private fun jumpToMethod() {
+    }
+
+    // For a fast jump to the place where you jus modified a code,
```
Comment (kahakai) [inline src/main/kotlin/day_1_kotlin/langeng/info/D_HowToUseHotkeys.kt]:
```suggestion
    // For a fast jump to the place where you just modified the code,
```

File: src/main/kotlin/day_1_kotlin/langrus/assignments/KotlinWorkshop3.kt [hunk 2]
```diff
@@ -23,80 +24,84 @@ object KotlinWorkshop3 {
...
     // TODO 5
-    // Should count and print guesses that were lower than randomNumber.
     val countLowerGuesses: (IntArray, Int) -> Unit = { guesses, randomNumber ->
         TODO()
     }
 
 
 
-    /* DO NOT TOUCH the utils below. */
+    /* Для корректного прогона воркшопа, не модифицируй утилиты ниже */
```
Comment (kahakai) [inline src/main/kotlin/day_1_kotlin/langrus/assignments/KotlinWorkshop3.kt]:
```suggestion
    /* Для корректного прогона воркшопа не модифицируй утилиты ниже */
```

File: src/main/kotlin/day_1_kotlin/langrus/info/D_HowToUseHotkeys.kt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    * Intellij idea and Android studio have a lot of templates, shortcuts and hot keys.
+    * And is very flexible and customizable. There are several of them:
+    * */
+
+    // For a fast jump to a method or variable,
+    // place a cursor on it and press "Ctrl+B"
+    private fun jumpToMethod() {
+    }
+
+    // For a fast jump to the place where you jus modified a code,
```
Comment (kahakai) [inline src/main/kotlin/day_1_kotlin/langrus/info/D_HowToUseHotkeys.kt]:
```suggestion
    // For a fast jump to the place where you just modified the code,
```

File: src/main/kotlin/day_1_kotlin/langrus/solutions/KotlinWorkshop3.kt [hunk 4]
```diff
@@ -131,7 +134,7 @@ object KotlinWorkshop3 {
 
 
 
-    /* DO NOT TOUCH the utils below. */
+    /* Для корректного прогона воркшопа, не модифицируй утилиты ниже */
```
Comment (kahakai) [inline src/main/kotlin/day_1_kotlin/langrus/solutions/KotlinWorkshop3.kt]:
```suggestion
    /* Для корректного прогона воркшопа не модифицируй утилиты ниже */
```

**Review by kahakai — APPROVED** [review]
It will be a good idea to include macOS shortcuts and hot keys.

File: README.md [hunk 5]
```diff
@@ -7,3 +7,5 @@ Package `assignments` contains tasks divided in workshop files.
 Package `solutions` contains... solutions! :)
 
 Package `info` contains simple help for students: how to comment, run and print code.
+
+Packages grouped by languages: `langeng` contains workshops with description in english, `langrus` in russian. Content is similar.
```
Comment (kahakai) [inline README.md]:
```suggestion
Packages grouped by languages: `langeng` contains workshops with description in English, `langrus` in Russian. Content is similar.
```

**Review by kahakai — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (webanimal) [reply README.md]: Fixed.

File: src/main/kotlin/day_1_kotlin/langrus/info/D_HowToUseHotkeys.kt (hunk 3 above)
Comment (webanimal) [reply src/main/kotlin/day_1_kotlin/langrus/info/D_HowToUseHotkeys.kt]: Fixed.

File: src/main/kotlin/day_1_kotlin/langrus/solutions/KotlinWorkshop3.kt (hunk 4 above)
Comment (webanimal) [reply src/main/kotlin/day_1_kotlin/langrus/solutions/KotlinWorkshop3.kt]: Fixed.

File: src/main/kotlin/day_1_kotlin/langrus/assignments/KotlinWorkshop3.kt (hunk 2 above)
Comment (webanimal) [reply src/main/kotlin/day_1_kotlin/langrus/assignments/KotlinWorkshop3.kt]: Fixed.

File: src/main/kotlin/day_1_kotlin/langeng/info/D_HowToUseHotkeys.kt (hunk 1 above)
Comment (webanimal) [reply src/main/kotlin/day_1_kotlin/langeng/info/D_HowToUseHotkeys.kt]: Fixed.

**Pull request merged**